### ISSUE TITLE
error in running jupyter in singularity

### ISSUE THREAD

**Issue by apal6** [apptainer/apptainer#1638]
Hi,

I am trying to run a jupyter notebook in singularity on HPC but i am not able to connect with the HPC remote host. I have the path with the token to be loaded on server but it fails. Not sure what might be going wrong. 
Any tips on how to debug? 

Thanks!

**Comment by GodloveD** [comment]
Are you port-forwarding from your local machine to the login node and from the login node to the compute node?

**Comment by apal6** [comment]
Yes, I am. After logging in the computing node, I am trying to run jupyter in singularity. I use the URL with the token to connect to the local host but it fails. Is it due to missing port?

**Comment by GodloveD** [comment]
Can you be more specific?  When you say "it fails" does this mean it gives you a 404 or a 401 or a black screen or something else?  The port on the compute node should be just fine since Apptainer is not running in a network namespace.  Just to be crystal clear you are doing something like this right?
```
$ ssh -L 8000:localhost:8000 demouser@$IP
```
You are doing this from your local machine to the login node and from the login node to the compute node right?
